MaxText/configs/base.yml

```diff
@@ -57,7 +57,7 @@ logits_via_embedding: True  # NOTE: this is True just for testing.
 remat_policy: 'full'
 scan_layers: True
 param_scan_axis: 1
-enable_flash_attention: True
```

Comment:
Have you `git grep enable_flash_attention` to make sure you aren't breaking anyone / updated everyone in this repo?